subworkflows/local/fastq_align_bwamem_mem2_dragmap_sentieon/main.nf

```diff
@@ -36,8 +36,7 @@ workflow FASTQ_ALIGN_BWAMEM_MEM2_DRAGMAP_SENTIEON {
     bam = bam.mix(DRAGMAP_ALIGN.out.bam)
     bam = bam.mix(SENTIEON_BWAMEM.out.bam_and_bai.map{ meta, bam, bai -> [ meta, bam ] })
 
-    bai = Channel.empty()
-    bai = bai.mix(SENTIEON_BWAMEM.out.bam_and_bai.map{ meta, bam, bai -> [ meta, bai ] })
+    bai = SENTIEON_BWAMEM.out.bam_and_bai.map{ meta, bam, bai -> [ meta, bai ] }
```

Comment:
I just copied the `Channel.empty()` from the definition of the bam-channel above. Why is it that that way of doing it is okay for the bam-channel but not for the bai-channel?

Reply:
it's ok, it just doesn't do anything in this case, you create an empty channel and then populate it with one element. In the case before we keep adding bams from the other mappers, which is not necessary here because of how sentieon vs gatk for md works. It just a matter of taste

Reply:
I see. And `bai` will still be properly defined in the case where `SENTIEON_BWAMEM` doesn't run because one of the other aligners was selected?

Reply:
yep tests are passing :) (not here but locally) I am not very fussed about it either way, just went over everything and just adapted stuff all over the place to have it a bit coherent

Reply:
All good. Thanks for improving the code quality and readability. I learn a bit from seeing how you do stuff ;-)